pkg/core/jsonschema/main.go

```diff
@@ -27,7 +27,7 @@ var (
 	updatecliPackageName string = "github.com/updatecli/updatecli/"
 
 	// commentDir defines the temporary directory where
```

Comment:
where what? ^^

Reply:
Thanks for catching this, I guess I delete this line somehow